autogluon/utils/tabular/features/auto_ml_feature_generator.py

```diff
@@ -222,3 +223,63 @@ def generate_text_ngrams(self, X, features_nlp_current, downsample_ratio: int =
             X_nlp_features_combined = pd.concat(X_nlp_features_combined, axis=1)
 
         return X_nlp_features_combined
+
+    def generate_text_special(self, X: Series, feature: str) -> DataFrame:
```

Comment:
could be nice to document this so interested users have an idea of the types of preprocessing applied.

Reply:
Agreed, this is planned to be done alongside the feature generation refactoring effort